src/css/application.css

```diff
@@ -473,6 +473,20 @@ body {
   cursor: pointer;
 }
 
+.errorList-error-message {
+  overflow-wrap: break-word;
+  word-wrap: break-word;
```

Comment:
What are your thoughts on *just* using `word-wrap`? As I understand it, `word-wrap` is officially considered an alias for `overflow-wrap`, but has better browser compatibility. Better to keep it simple and just use `word-wrap`, which is widely supported?